ENGAGEHF/Medications/Medications.swift

```diff
@@ -0,0 +1,23 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+struct Medications: View {
+    @Binding var presentingAccount: Bool
+    
+    
+    var body: some View {
+        Text("Medications go here.")
```

Comment:
We should add the account button here as well, we already have the binding in place here.